core/src/main/java/hudson/util/HudsonIsLoading.java

```diff
@@ -41,7 +41,7 @@
  */
 public class HudsonIsLoading {
     public void doDynamic(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException, InterruptedException {
-        rsp.setStatus(SC_SERVICE_UNAVAILABLE);
+        rsp.setStatus(HttpURLConnection.HTTP_UNAVAILABLE);
```

Comment:
Both are 503, what am I missing?

Reply:
> The change in the production code is just to reduce the use of javax package when it's easy.

I think he means this. Not certain of the motivation for it.

Reply:
> The change in the production code is just to reduce the use of javax package when it's easy.

I think he means this. Not certain of the motivation for it.

Reply:
It's exactly that. I changed them just because of the license problem with javax stuff IIRC, from Stephen about the `@Nonnull` or similar stuff.

Reply:
That was specifically for those nullability annotations only: They're not actually provided by the Java platform, but an independent implementation, and that's not supposed to go into `javax`.

Reply:
Reverted :)